File: Team C (Season 2)/Week 1/Greed/86491. 최소직사각형/최소직사각형.swift
import Foundation

func solution(_ sizes:[[Int]]) -> Int {
    var maxNum = 0
    var minNum = 0
    
    for size in sizes {
        maxNum = max(maxNum, size.max()!)
        minNum = max(minNum, size.min()!)
    }
    return maxNum * minNum
}

File: Team C (Season 2)/Week 1/Greed/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    answer = 0
    maxW = 0
    maxH = 0
    
    for w,h in sizes:
        if h > w:
            w,h = h,w
        maxW = max(w,maxW)
        maxH = max(h,maxH)
        
    answer = maxW * maxH
    return answer